internal/perms/perms.go

```diff
@@ -17,35 +17,21 @@ package perms
 
 import (
 	"fmt"
-	"os/user"
-	"strconv"
+	"os"
 )
 
 // MyUserAndGroup returns the UID and GID of this process.
 func MyUserAndGroup() (uid uint32, gid uint32, err error) {
-	// Ask for the current user.
-	user, err := user.Current()
-	if err != nil {
-		err = fmt.Errorf("Fetching current user: %w", err)
-		return
-	}
-
-	// Parse UID.
-	uid64, err := strconv.ParseUint(user.Uid, 10, 32)
-	if err != nil {
-		err = fmt.Errorf("Parsing UID (%s): %w", user.Uid, err)
-		return
-	}
+	signed_uid := os.Getuid()
+	signed_gid := os.Getgid()
```

Comment:
Did we compare the value of new methods vs old methods. Does it return the same uid/gid?